Gems/Multiplayer/Code/Source/MultiplayerSystemComponent.cpp

```diff
@@ -624,6 +624,15 @@ namespace Multiplayer
             }
         }
 
+        if (static_cast<AZ::CVarFixedString>(sv_map).empty())
+        {
+            AZLOG_WARN("Server does not have a multiplayer level loaded! Make sure the server has a level loaded before accepting clients.");
+            m_noServerLevelLoadedEvent.Signal();
+
+            connection->Disconnect(DisconnectReason::ServerNoLevelLoaded, TerminationEndpoint::Local);
+            return true;
+        }
+
```

Comment:
Does this work with the Multiplayer Editor UX given the Editor tells the Server what to load in that specific case?